frontend/src/app/home/home.component.html

```diff
@@ -1,4 +1,134 @@
-<h1>Home</h1>
-<p class="h3">Welcome to the Merovingian Lettercraft project!</p>
-<p>This website is still under construction. Please check back again in the future.</p>
+<div class="home-background pt-5">
+    <header>
+        <div class="container vh-100">
+            <div class="row d-flex justify-content-end h-75 align-items-end">
+                <div class="col-lg-6 card">
+                    <div class="card-body py-5 px-3 text-center">
+                        <h1 class="display-4 text-primary mb-3">
+                            <span class="illuminated-initial">L</span>ettercraft
+                        </h1>
+                        <p class="lead">
+                            'Lettercraft and Epistolary Performance in Early
+                            Medieval Europe, 476-751 CE' is an NWO-Vidi funded
+                            project based in the Department of Languages,
+                            Literature, and Culture at Utrecht University.
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
 
+    <section class="banner bg-primary text-secondary p-5 mb-5 lh-lg border-y-gold">
+        <h2 class="text-center mb-4">Welcome!</h2>
+
+        <div class="banner-content">
```

Comment:
Because this is a longer piece of text, I would use the gold colour for the header but make the body text white or ecru, for more comfortable reading.